### DISCUSSION TITLE
Github rollover popups are hanging

### DISCUSSION THREAD

**Discussion by rgleason** [community/community#44674] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Does anyone have trouble with mouse rollover popups which are hanging up and not disappearing?
How does one prevent that or make it actually work better?

The problem is I cannot get it to go away, and in any case that is not what I am interested in.
I don't want to have to click or keystroke it to hide.
I just want to get to something that is behind that window or read something behind it.

It is very frustrating.  Windows 11    Firefox Browser

**Comment by jules32** [comment]
I'm also experiencing this; Mac 12.5; Firefox Browser

**Reply by WebMechanic** [reply]
dito in all Firefox Win10. 
Super annoying feature! 
They popup everywhere one hovers unexpectedly: navigate someplace else, and if a username or link happens to be under the mouse pointer on the new page and: bang in your face.
They can't be "dismissed" with ESC or _any_ click-outside.

https://www.nngroup.com/articles/tooltip-guidelines/ (Popup tips)

**Comment by queenofcorgis** [comment]
Hi there 👋🏼  - I have passed this on to the proper team - they are aware that this is affecting users in the Firefox browser and working to troubleshoot. When I have updates I'll share them with you.

**Reply by kdahlquist** [reply]
Thank you for working on this--it is very annoying!

**Reply by WebMechanic** [reply]
@queenofcorgis is there a public issue URL you could post , so we can follow that?

**Reply by queenofcorgis** [reply]
Hi there @WebMechanic - unfortunately this work is taking place in an Issue in a private repo. But it looks like we have a fairly promising fix in the works - thread [here](https://github.com/orgs/community/discussions/44630#discussioncomment-4833394) for more information and updates.

**Comment by rgleason** [comment]
This hasn't really changed. Still and issue.

**Comment by WebMechanic** [comment]
there appears to be some change (FF 110/Win) but the hover now is random: If you hover over them several times (+3) they stop working.
At least they disappear if you move off the links

**Comment by radar-blip** [comment]
Seems to be fixed now. 😃

**Comment by WebMechanic** [comment]
"fixed" as in the do go away, so yes: 🎉 

as a feature still somewhat broken 'cos they no longer work on internal links like issues, comments, branches (which I don't mind) and randomly don't work for usernames, especially after you previously hovered several times over any
